test-utils/src/spl.rs

```diff
@@ -41,7 +42,7 @@ impl MintFixture {
                 Mint::LEN as u64,
                 &spl_token::id(),
             );
-            let init_mint_ix = initialize_mint(
+            let init_mint_ix = spl_token_2022::instruction::initialize_mint(
```

Comment:
Shouldn't this remain tokenkeg's `initialize_mint`? Since there is a separate builder for t22 mint fixtures.
Or does token22 simply not affect the Mint and those fns are equivalent?

Reply:
The instruction is the same, just the program address is different.